Components/AppTabNavigator/StocksTab.js

```diff
@@ -23,16 +23,76 @@ class StocksTab extends Component {
         companyNames: [],
         companySymbols: [],
       },
+      favoritedCompanies: {
+        companyNames: [],
+        companySymbols: [],
+      },
     };
   }
 
   componentDidMount() {
-    //NetInfo.isConnected.fetch().then(isConnected => {
     if (true) {
```

Comment:
Does this serve a purpose? is it like the isMounted = true check

Reply:
I was also wondering the same thing. I don't think I was the one to put in this if statement but I'm gonna test without the if statement and remove it if it's unnecessary.